### ISSUE TITLE
[Question] video_player_media_kit 1.0.2 There is picture but no sound when playing video

### ISSUE THREAD

**Issue by Fantuan-cell** [media-kit/media-kit#522]
dependencies:
```  
video_player_media_kit: ^1.0.2
  media_kit_libs_android_video: any
  media_kit_libs_ios_video: any #ios
```
There is no sound when playing videos using video_player_media_kit. What needs to be set?

mycode
```
void main() async {

  VideoPlayerMediaKit.ensureInitialized(
    android: true,
    iOS: true,
  );
  runApp(const AppPage());
}

```

![image](https://github.com/media-kit/media-kit/assets/53990888/0412afc5-f439-4a36-9e37-38e660e3df3b)

**Comment by alexmercerind** [comment]
Please share sample file/URL.

**Comment by Fantuan-cell** [comment]
> Please share sample file/URL.
> 
> You may mail me at <[alexmercerind@gmail.com](mailto:alexmercerind@gmail.com)>.

https://vod6.bdzybf7.com/20231006/8Rnuv3uX/index.m3u8

https://vkceyugu.cdn.bspapp.com/VKCEYUGU-ce0dc77e-96a6-43bc-b253-359189a82422/feb8aa30-56c4-49fb-bed9-1307f007ba9c.mp4

**Comment by Fantuan-cell** [comment]
I am using a real phone and the system is Android 8.0

**Comment by Fantuan-cell** [comment]
My phone directly uses the Video_player plugin to play videos, and there is sound and picture.

**Comment by Fantuan-cell** [comment]
> My phone directly uses the Video_player plugin to play videos, and there is sound and picture.

But when using video_player_media_kit to play videos, there will be no sound.

**Comment by alexmercerind** [comment]
Does our default media_kit test application work well?

You can download pre-built APK from our README:

https://github.com/media-kit/media-kit#platforms

**Comment by alexmercerind** [comment]
We use OpenSL ES audio output driver by default.

**Comment by alexmercerind** [comment]
Hi!

Your feedback will be helpful. We're committed to resolve your issue.

**Comment by alexmercerind** [comment]
Hi @Fantuan-cell.

I updated media_kit_libs_android_video in recently; you may test if your issue is resolved.

Also: Does [mpv-android](https://play.google.com/store/apps/details?id=is.xyz.mpv) work on your device?

**Comment by longgong** [comment]
I also have similar problems， no sound in windows。

**Comment by longgong** [comment]
@alexmercerind   There is no sound even when using the demo you provided.

**Comment by zezo357** [comment]
@longgong do you call `setVolume` anywhere in your code? i think that was the problem 
check fix in: https://github.com/media-kit/media-kit/pull/598, 

you test it by adding in pubspec 

```
dependency_overrides:
  video_player_media_kit:
    git:
      url: https://github.com/zezo357/media_kit
      path: ./video_player_media_kit
      ref: fix-sound
```

and let me know if fixed the issue for you so we merge it

**Comment by longgong** [comment]
> @longgong do you call `setVolume` anywhere in your code? i think that was the problem check fix in: #598,
> 
> you can test it by adding in pubspec
> 
> ```
> dependency_overrides:
>   video_player_media_kit:
>     git:
>       url: https://github.com/zezo357/media_kit
>       path: ./video_player_media_kit
>       ref: fix-sound
> ```
> 
> and let me know if fixed the issue for you so we merge it

yes，I just saw that modification. There is sound now.

**Comment by zezo357** [comment]
Glad to hear that, I think this issue need to be reopened @alexmercerind until it's merged so I will reopen it.
